systems.csv

```diff
@@ -191,13 +198,16 @@ ES,BBK Klimabizi,"Urdaibai, ES",nextbike_ey,https://www.nextbike.es/es/bbkklima/
 ES,Bicing,"Barcelona, ES",bike_barcelona,https://www.bicing.barcelona/,https://barcelona-sp.publicbikesystem.net/customer/gbfs/v2/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fbarcelona-sp.publicbikesystem.net%2Fcustomer%2Fgbfs%2Fv2%2Fgbfs.json
+ES,BiciPalma,"Palma, ES",nextbike_ea,https://www.bicipalma.com/es/palma/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ea/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ea%2Fgbfs.json
 ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eus/eu/bilbao/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_bo/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_bo%2Fgbfs.json
 ES,Bird Madrid,"Madrid, ES",bird-madrid,https://www.bird.co,https://mds.bird.co/gbfs/v2/public/madrid/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fmds.bird.co%2Fgbfs%2Fv2%2Fpublic%2Fmadrid%2Fgbfs.json
 ES,Bird Zaragoza,"Zaragoza, ES",bird zaragoza,https://www.bird.co,https://mds.bird.co/gbfs/v2/public/zaragoza/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fmds.bird.co%2Fgbfs%2Fv2%2Fpublic%2Fzaragoza%2Fgbfs.json
 ES,Canary Islands,"Canary Islands, ES",Link_Canary_Islands,https://www.link.city,https://mds.linkyour.city/gbfs/es_canary_islands/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fmds.linkyour.city%2Fgbfs%2Fes_canary_islands%2Fgbfs.json
 ES,Dbizi,"Donostia, ES",sansebastian,https://www.dbizi.eus,https://sansebastian.publicbikesystem.net/ube/gbfs/v1/,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fsansebastian.publicbikesystem.net%2Fube%2Fgbfs%2Fv1%2F
 ES,Donkey Republic Barcelona,"Barcelona, ES",donkey_barcelona,https://www.donkey.bike/cities/bike-rental-barcelona/,https://stables.donkey.bike/api/public/gbfs/2/donkey_barcelona/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fstables.donkey.bike%2Fapi%2Fpublic%2Fgbfs%2F2%2Fdonkey_barcelona%2Fgbfs.json
 ES,El Puerto de Santa Maria,"El Puerto de Santa Maria, ES",Link_El_Puerto_de_Santa_Maria,https://www.link.city,https://mds.linkyour.city/gbfs/es_el_puerto_de_santa_maria/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fmds.linkyour.city%2Fgbfs%2Fes_el_puerto_de_santa_maria%2Fgbfs.json
+ES,Getxobizi,"Getxo, ES",nextbike_ex,https://www.nextbike.es/es/getxo/eu/getxo/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ex/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ex%2Fgbfs.json
+ES,Granada,"Granada, ES",Link_Granada,https://superpedestrian.com/,https://mds.linkyour.city/gbfs/es_granada/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fmds.linkyour.city%2Fgbfs%2Fes_granada%2Fgbfs.json
```

Comment:
question (non-blocking): it's unclear to me why https://mds.linkyour.city/gbfs/es_granada/gbfs.json is returning no results on the validator. This is probably my own ignorance. Could you share context?

Reply:
Seems to be an issue with the user agent, an [issue](https://github.com/MobilityData/gbfs-validator/issues/90#issuecomment-1411682286) has been opened on the validator to address it.